Antidote/Images.xcassets/Controllers/Call/phone.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x",
+      "filename" : "phone.png"
```

Comment:
Please use more meaningful names for images (you can check other images). Also there should be `@2x` suffix for 2x images

 `call-phone@2x.png`